pyhub/mcptools/google/sheets/tools/__init__.py

```
@@ -0,0 +1,3 @@
+"""Google Sheets MCP Äl"""
```

Ruff (0.11.9)

stream did not contain valid UTF-8

(E902)